.github/workflows/npm-publish_stage.yml

```
@@ -8,18 +8,18 @@ on:
-    contents: write
+  contents: write
 
 jobs:
   build-javascript-client:
     runs-on: ubuntu-latest
 
     steps:
       - uses: actions/checkout@v3
-      - name: Use Node.js 16.x
+      - name: Use Node.js 18.x
         uses: actions/setup-node@v3
```

actionlint (1.7.4)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)